content/docs/hello-world.md

```diff
@@ -1,50 +1,50 @@
 ---
 id: hello-world
-title: Hello World
+title: Helló, világ!
 permalink: docs/hello-world.html
 prev: cdn-links.html
 next: introducing-jsx.html
 ---
 
-The smallest React example looks like this:
+A legkisebb React példa így néz ki:
 
 ```js
 ReactDOM.render(
-  <h1>Hello, world!</h1>,
+  <h1>Helló, világ!</h1>,
   document.getElementById('root')
 );
 ```
 
-It displays a heading saying "Hello, world!" on the page.
+Ez egy címsort jelenít meg "Hello, világ!" szöveggel a weboldalon.
 
 [](codepen://hello-world)
 
-Click the link above to open an online editor. Feel free to make some changes, and see how they affect the output. Most pages in this guide will have editable examples like this one.
+Kattints a fenti hivatkozásra az online szerkesztő megnyitásához. Bátran végezz változtatásokat, és figyeld meg hogyan befolyásolják a kimenetet. Az útmutató legtöbb oldala ehhez hasonló szerkeszthető példákat tartalmaz.
 
 
-## How to Read This Guide {#how-to-read-this-guide}
+## Hogyan használd helyesen az útmutatót {#how-to-read-this-guide}
 
-In this guide, we will examine the building blocks of React apps: elements and components. Once you master them, you can create complex apps from small reusable pieces.
+Ebben az útmutatóban a React alkalmazások építőkockáit fogjuk megvizsgálni: elemeket és komponenseket. Miután elsajátítottad használatukat, összetett alkalmazásokat tudsz majd létrehozni kis újrafelhasználható részekből.
 
->Tip
+>Tipp
 >
->This guide is designed for people who prefer **learning concepts step by step**. If you prefer to learn by doing, check out our [practical tutorial](/tutorial/tutorial.html). You might find this guide and the tutorial complementary to each other.
+>Ez az útmutató olyan emberek számárak készült, akik előnyben részesítik a **lépésről lépésre történő tanulást**. Ha jobban szereted gyakorlati példák közben megszerezni a tudást, nézd meg a [tutoriált](/tutorial/tutorial.html). Ezt az útmutatót és a tutoriált egymás kiegészítésének is tekintheted.
```

Comment:
```suggestion
>Ez az útmutató azok számára készült, akik előnyben részesítik a **koncepciók lépésről lépésre történő tanulást**. Ha jobban szereted gyakorlati példák közben megszerezni a tudást, nézd meg a [gyakorlati tutoriált](/tutorial/tutorial.html). Ezt az útmutatót és a tutoriált egymás kiegészítésének is tekintheted.
```

Reply:
> Ez az útmutató azok számára készült, akik előnyben részesítik a **koncepciók lépésről lépésre történő tanulást**. 

Ez így teljesen nem értelmes, első mondat vége lehetne "tanulását".
Ezt tudod módosítani légyszíves, utánna akkor ezt is commitálom és tudom mergelni

Reply:
Elírás volt :angel: